### PR TITLE
[MODORDERS-902] - Change karate test in a way that it should be

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
Karate test is failing
## Approach
- Karate test was fixed based on comment
image
image

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure headers = headersUser
+
+    * callonce variables
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+
+  Scenario:  Prepare finances
```
Comment (Saba-Zedginidze-EPAM) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature:19]: Probably it is better to do all data initialization in Background in general, so Scenarios are not dependent on each other

**Review by Saba-Zedginidze-EPAM — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)'}
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)'}
+
+
+  Scenario: Verify holding NOT being created when createInventory: None
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature:25]: Can add @Positive to all scenarios, neither of them fail controllably so no @Negative is needed

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature (hunk 1 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature]: Fixed it, thanks

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature (hunk 2 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature:25]: Added, thanks

**Pull request merged**